scrapy/utils/signal.py

```diff
@@ -97,7 +98,10 @@ def logerror(failure: Failure, recv: Any) -> Failure:
             robustApply, receiver, signal=signal, sender=sender, *arguments, **named
         )
         d.addErrback(logerror, receiver)
-        d.addBoth(lambda result: (receiver, result))
+        # TODO https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/cell-var-from-loop.html
```

Comment:
The article suggests that the code doesn't work the way it looks, we should investigate if this indeed applies to our code (I would be surprised that signals are sent to wrong places as we would notice that).